src/AdScreenerTool.js

```diff
@@ -227,6 +235,17 @@ const AdScreener = (params) => {
     getAds();
   };
 
+  const handleSearch = (k) => {
```

Comment:
to me `handleSearch` sounds like this is dispatching a call to a search backend. But this appears to be handling select on the topic and full text search tabls. First I think we need name for these tabs (naming is hard and I can think of right now is topicOrFullTextSearchTabs), and then this function can be handleSelect<name for these tabs>. What do you think?

Reply:
I can change the fucntion name to handleSelect. But what do you mean by naming the tabs ? Do you want me to create a new component ?

Reply:
It doesn't need to be a new component. I was just trying to think of what it would be named if it were component. So that we have a way to refer to it and/or a common name to use in vars/functions related to it.